locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -914,6 +914,7 @@ msgid "Question"
 msgstr ""
 
 #: corehq/apps/app_manager/static/app_manager/js/summary/models.js
+#: corehq/apps/app_manager/static/app_manager/js/vellum/src/main-components.js
```

Comment:
I assume the changes in this directory are simply because of changes since Wednesday that we haven't updated translations for yet? Just double checking my understanding 😄

Reply:
Yes. Your understanding is correct. These are the changes that happened on HQ code between Wednesday(when translations were merged) and yesterday when I ran make messages again.